### PR TITLE
feat: support lazy modifier with setValue

### LINKED ISSUE #1465: setValue does not apply changes to model when lazy modifier is present
### Version
1.0.0-beta.32

### Reproduction link
[https://github.com/jaredmcateer/vtu-set-value](https://github.com/jaredmcateer/vtu-set-value)

### Steps to reproduce
Clone the minimal test case repo, run `npm install` once that is complete run `node test.js` The assert will fail on line 35 as the model has not updated. Commenting out line 35 will allow the remaining assert to run, which will pass because the change event has been triggered.

### What is expected?
When calling setValue the model should be updated to reflect that value

### What is actually happening?
setValue triggers an "input" event on text input/textareas. A model with the lazy modifier does not update the model until the change event is triggered (see: https://vuejs.org/v2/guide/forms.html#lazy) 

---
If the desire is for setValue to mimic typing perhaps then the documentation should indicate that setValue must be accompanied by a triggering of the change event if the lazy modifier is present or an optional parameter could be passed to setValue to force a change event.

### PR DESCRIPTION
**Pull request by lmiller1990**
Fixes #1465

### CODE REVIEW

File: packages/test-utils/src/wrapper.js [hunk 1]
```diff
@@ -569,10 +569,19 @@ export default class Wrapper implements BaseWrapper {
...
-      this.trigger(event)
+
+      if (tagName === 'SELECT') {
+        this.trigger('change')
+      } else {
+        this.trigger('input')
+      }
+
+      // for v-model.lazy, we need to trigger a change event, too.
+      if (tagName === 'INPUT' && this.element._vModifiers) {
```
Comment (dobromir-hristov) [inline packages/test-utils/src/wrapper.js]: shall we do `vModifiers.lazy` ?

Comment (dobromir-hristov) [reply packages/test-utils/src/wrapper.js]: Lazy can be used on textarea too :)

**Review by jaredmcateer — COMMENTED** [review]
Other than @dobromir-hristov's review points this is a satisfactory solution for me. Thanks for the quick turnaround!

File: packages/test-utils/src/wrapper.js (hunk 1 above)
Comment (lmiller1990) [reply packages/test-utils/src/wrapper.js]: Oops, I thought that was what I was doing. Yes, we definitely should, I'll fix this.

Comment (lmiller1990) [reply packages/test-utils/src/wrapper.js]: Fixed

File: packages/test-utils/src/wrapper.js [hunk 2]
```diff
@@ -569,10 +569,24 @@ export default class Wrapper implements BaseWrapper {
...
+      if (tagName === 'SELECT') {
+        this.trigger('change')
+      } else {
+        this.trigger('input')
+      }
+
+      // for v-model.lazy, we need to trigger a change event, too.
+      // $FlowIgnore
+      if (
+        tagName === 'INPUT' &&
```
Comment (dobromir-hristov) [inline packages/test-utils/src/wrapper.js]: `.lazy` can also be used on `TEXTAREA`.

**Comment by lmiller1990** [comment]
passes locally - why is CI failing? 🤔

File: packages/test-utils/src/wrapper.js (hunk 2 above)
Comment (lmiller1990) [reply packages/test-utils/src/wrapper.js]: oops, updated! all green locally. CI failing though 🤔

**Comment by lmiller1990** [comment]
oh, I guess lazy isn't in old versions of Vue. I'll fix that.

**Comment by lmiller1990** [comment]
oh, I guess lazy isn't in old versions of Vue. I'll fix that.

**Pull request merged**